actioncable/blade.yml

```diff
@@ -5,3 +5,9 @@ load_paths:
 
 logical_paths:
   - test.js
+
+build:
+  logical_paths:
+    - action_cable.js
+  path: lib/assets/compiled
+  js_compressor: uglifier
```

Comment:
Ah, right. Forgot we only include `lib` in the gem. 

Either way, I'm not sure your use case of editing an installed gem justifies distributing a larger than necessary .js file.